taskcluster/tc-cppwin_tflite-ds-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This file is identical to `taskcluster/tc-cpp_tflite-ds-tests.sh`, why do we need it?

Reply:
No need